app/domain/errors.rb

```diff
@@ -360,7 +360,7 @@ module AuthnGcp
       InvalidAccountInAudienceClaim = ::Util::TrackableErrorClass.new(
         msg: "'audience' token claim '{0}' is invalid. " \
               "The account in the audience '{1}' does not match the account in the URL request '{2}'",
-        code: "CONJ00071E"
+        code: "CONJ00070E"
```

Comment:
@hilagross
How can we insure the deleted code not appears somewhere in the documentation?

Reply:
@tzheleznyak  - The account in audience '{1}' does not match the account in URL request '{2}'"